ElrondNetwork/elrond-go
# use elastic upsert instead of insert/update for transactions

Use `upsert` instead of different logic for `insert` and `update` for elastic search transactions.
This should solve cases when reimporting data would have caused a sender shard process a transaction after the receiver shard and update the status to 'pending'

Testing procedure:
1. run several nodes with import-db
2. transactions' statuses should not be 'pending'
